spec/tests/test.fmf

```diff
@@ -9,10 +9,8 @@ description: |
     Allows to parametrize a single test script and in this way
     create virtual test cases.
 
-    Special value ``manual`` can be used for tests which haven't
-    been automated yet. Such tests will be skipped during test
-    execution. In the future we might add support for executing
-    manual tests using user input to provide actual test results.
+    If test is manual (see manual attribute) it points to the document
```

Comment:
Use `:ref:'spec/tests/manual'` (with backquote) to directly reference the `manual` attribute documentation.